### ISSUE TITLE
Add syslog check in MyHTTPServer on Ubuntu in case of Chrome browser

### ISSUE THREAD

**Issue by AntoineGautier** [lbl-srg/funnel#42]
This is for BuildingsPy [#339](https://github.com/lbl-srg/BuildingsPy/issues/339).

**Comment by AntoineGautier** [comment]
@mwetter I could not reproduce the issue either on my virtual machine or on my Linux desktop. The patch I implemented in `issue42_syslogCheck` consists in:

- Monitoring `/var/log/syslog` for 1s after the browser is launched (I tried the avoid reading the whole file as it can be large so I only stream the new lines that are appended).
- Killing Chrome process in case `ERROR:gles2_cmd_decoder` is detected. This is quite brutal as it will close all existing browser windows, not only the ones opened by the module. The only other way that I can think of to terminate the guilty Chrome instance in the background will introduce significant new dependencies (namely Selenium and ChromeDriver) while the ability to fix the bug by just closing the latest open tab is more uncertain.
- Holding the execution and prompting the user to retry launching the browser.

I also packaged the code to make the installation easier and get rid of git submodule in BuildingsPy.

I tested the implementation by momentarily changing the error pattern to a standard info message that gets syslogged  when Chrome is launched. When you get the chance could you also test it on your system that showed the error? To do so: clone the repo, cd into it, and checkout `issue42_syslogCheck`.
Then
```
pip install .
pip install -r requirements.txt
cd tests/test_bin
```
Launch IPython and
```python
import time
from funnel import pyfunnel
time.sleep(20); pyfunnel.plot_funnel('results')
```
Then lock your screen within 20s and for more than 20s so that the browser is launched while the screen is indeed locked (make sure you have Chrome set as default). Might be good to check `/var/log/syslog` before and after that process to see if an error has been logged there.

**Comment by mwetter** [comment]
@AntoineGautier Thanks, I will test when I am at the lab again.

**Comment by mwetter** [comment]
@AntoineGautier : I run the test. After unlocking the screen I see chrome with the funnel plot. The file `/var/log/syslog` has no new entry.

If this is the expected behavior, can you please merge it to the `master` and update `BuildingsPy`.

**Comment by AntoineGautier** [comment]
Closed by #43.
